### ISSUE TITLE
Progress bar used to work but now doesn't appear anymore

### ISSUE THREAD

**Issue by ama12321** [carlos1172/ProgressBarTimeLeft#8]
Found your add on on reddit and loved it. 

One day, however, I somehow managed to click and move the progress bar so it was floating mid screen, and after restarting Anki it disappeared forever. It only has issues on one of my devices, the other device works fine. 

I'm currently on 2.1.54, and I just tried removing all add-ons except for this progress bar but it still doesn't show up (others said it might have interfering progress bars). 

Do you know if this is an HTML CSS thing? I'm bad at coding but I can't figure out how to get it back.

I have not tried completely deleting and reinstalling Anki but I have updated versions for Anki and this add on and neither have brought it back.

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/108439848/176568324-d1d2ac56-ef52-4b11-a432-561f70f7cb8f.png">

**Comment by carlos1172** [comment]
Have you tried reinstalling the add-on?

**Comment by carlos1172** [comment]
Have you tried reinstalling the addon?

**Comment by ama12321** [comment]
Yes, I have tried deleting and reinstalling the add on both through ankiweb
and github, but it still didn't work.

https://www.reddit.com/r/Anki/comments/r143qe/comment/hnsy9q2/?utm_source=share&utm_medium=web2x&context=3
This guy describes the problem I'm describing, not sure if you've been able
to figure it out.

https://postimg.cc/kRBTfVCS

**Comment by carlos1172** [comment]
I don’t know, sorry :(

Regards,
Carlos Resurreccion

**Comment by Vincedt1** [comment]
Hello,
Thanks for the work.
But exactly the same problem here. I uninstalled anki and tested different versions but it didn't solve the problem...

**Comment by ama12321** [comment]
No problem, thanks for the work you did! I really liked the add on because
it gave you a count of how many cards you've done while doing the cards and
not on the home page, which the original progress bar doesn't do. That is
the main feature I am sad I can't use.
